vnext/local-cli/generator-windows/index.js

```diff
@@ -81,6 +81,27 @@ function copyProjectTemplateAndReplace(
   const currentUser = username.sync(); // Gets the current username depending on the platform.
   const certificateThumbprint = generateCertificate(srcPath, destPath, newProjectName, currentUser);
 
+
+  const xamlNugetPkgName = options.useWinUI3 ? 'Microsoft.WinUI' : 'Microsoft.UI.Xaml';
```

Comment:
Are we really changing the root namespace?

Reply:
@jonthysell this line is for the name of the nuget package. Since the WinUI 3 package includes COMP in addition to XAML, the new package name is WinUI instead of MUX